src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,11 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+
+public class CandidateValidator implements Predicate<String> {
```

Comment:
```suggestion
public class CandidateValidator implements Predicate<Candidate> {
```
You shoud take `Candidate` here and do checks